src/pages/home/ReportScreen.js

```diff
@@ -73,6 +75,11 @@ class ReportScreen extends React.Component {
         this.loadingTimerId = setTimeout(() => this.setState({isLoading: false}), 300);
     }
 
+    storeCurrentlyViewedReport() {
+        const reportID = this.getReportID();
+        updateCurrentlyViewedReportID(reportID);
```

Comment:
We don't need an `if` guard here as currently it's guaranteed that `reportID` would be a proper value

This function would not run for the initial render but only for updates, which guarantees `reportID` would exist. For the initial render we either had something in Onyx or never had anything anyway.

Anyway I can add a safety check if you'd like